### PR TITLE
docs: add view user guide

### PR DESCRIPTION
**Pull request by killme2008**
## What's Changed in this PR

*Describe the change in this PR*

Adds view user guide. The next task is updating the references.

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.9/en/user-guide/query-data/view.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# View
+
+In SQL, a view is a virtual table based on the result set of an SQL statement. It contains rows and columns just like a real table.
```
Comment (nicecui) [inline docs/v0.9/en/user-guide/query-data/view.md:3]: Here we explained what a View is.
we need to further explain why users need a View:
- What problem does the View solve for users
- In what scenarios do users need to user View

**Review by nicecui — COMMENTED** [review]
almost LGTM

File: docs/v0.9/en/user-guide/query-data/view.md (hunk 1 above)
Comment (killme2008) [reply docs/v0.9/en/user-guide/query-data/view.md]: Done

File: docs/nightly/en/user-guide/query-data/view.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+* Granting read permissions to specific users while restricting access to certain columns and rows to ensure data security and isolation.
+
+A view is created with the `CREATE VIEW` statement.
+
+## View examples
+
+```sql
+CREATE VIEW cpu_monitor AS
+    SELECT cpu, host, ts FROM monitor;
+```
```
Comment (nicecui) [inline docs/nightly/en/user-guide/query-data/view.md:15]: when new data is inserted into the `monitor`, will the data in the `cpu_monitor` view be updated synchronously?

Comment (killme2008) [reply docs/nightly/en/user-guide/query-data/view.md:15]: There is no update happening, view is just a represent of a query.

**Review by nicecui — APPROVED** [review]

File: docs/nightly/zh/user-guide/query-data/view.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+|  0.3 | 127.0.0.2 | 2023-12-13 02:05:41 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:46 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+通过 `WHERE` 查询视图：
+
+```sql
+SELECT * FROM cpu_monitor where host = '127.0.0.2';
```
Comment (tisonkun) [inline docs/nightly/zh/user-guide/query-data/view.md:44]:
```suggestion
SELECT * FROM cpu_monitor WHERE host = '127.0.0.2';
```

File: docs/nightly/zh/user-guide/query-data/view.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+创建一个从两个表联合查询数据的视图：
+
+```sql
+CREATE VIEW app_cpu_monitor AS
+    SELECT cpu, latency, host, ts FROM monitor LEFT JOIN app_monitor
+    ON monitor.host = app_monitor.host and monitor.ts = app_monitor.ts
```
Comment (tisonkun) [inline docs/nightly/zh/user-guide/query-data/view.md:64]:
```suggestion
    ON monitor.host = app_monitor.host AND monitor.ts = app_monitor.ts
```

File: docs/nightly/en/user-guide/query-data/view.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+|  0.3 | 127.0.0.2 | 2023-12-13 02:05:41 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:46 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+Query view by `WHERE`:
+
+```sql
+SELECT * FROM cpu_monitor where host = '127.0.0.2';
```
Comment (tisonkun) [inline docs/nightly/en/user-guide/query-data/view.md:43]:
```suggestion
SELECT * FROM cpu_monitor WHERE host = '127.0.0.2';
```

File: docs/nightly/en/user-guide/query-data/view.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+Create a view that queries data from two tables:
+
+```sql
+CREATE VIEW app_cpu_monitor AS
+    SELECT cpu, latency, host, ts FROM monitor LEFT JOIN app_monitor
+    ON monitor.host = app_monitor.host and monitor.ts = app_monitor.ts
```
Comment (tisonkun) [inline docs/nightly/en/user-guide/query-data/view.md:61]:
```suggestion
    ON monitor.host = app_monitor.host AND monitor.ts = app_monitor.ts
```

File: docs/v0.9/zh/user-guide/query-data/view.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+|  0.3 | 127.0.0.2 | 2023-12-13 02:05:41 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:46 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+通过 `WHERE` 查询视图：
+
+```sql
+SELECT * FROM cpu_monitor where host = '127.0.0.2';
```
Comment (tisonkun) [inline docs/v0.9/zh/user-guide/query-data/view.md:44]:
```suggestion
SELECT * FROM cpu_monitor WHERE host = '127.0.0.2';
```

File: docs/v0.9/zh/user-guide/query-data/view.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+创建一个从两个表联合查询数据的视图：
+
+```sql
+CREATE VIEW app_cpu_monitor AS
+    SELECT cpu, latency, host, ts FROM monitor LEFT JOIN app_monitor
+    ON monitor.host = app_monitor.host and monitor.ts = app_monitor.ts
```
Comment (tisonkun) [inline docs/v0.9/zh/user-guide/query-data/view.md:64]:
```suggestion
    ON monitor.host = app_monitor.host AND monitor.ts = app_monitor.ts
```

File: docs/v0.9/en/user-guide/query-data/view.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+|  0.3 | 127.0.0.2 | 2023-12-13 02:05:41 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:46 |
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+Query view by `WHERE`:
+
+```sql
+SELECT * FROM cpu_monitor where host = '127.0.0.2';
```
Comment (tisonkun) [inline docs/v0.9/en/user-guide/query-data/view.md:43]:
```suggestion
SELECT * FROM cpu_monitor WHERE host = '127.0.0.2';
```

File: docs/v0.9/en/user-guide/query-data/view.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+|  0.2 | 127.0.0.2 | 2023-12-13 02:05:51 |
++------+-----------+---------------------+
+```
+
+Create a view that queries data from two tables:
+
+```sql
+CREATE VIEW app_cpu_monitor AS
+    SELECT cpu, latency, host, ts FROM monitor LEFT JOIN app_monitor
+    ON monitor.host = app_monitor.host and monitor.ts = app_monitor.ts
```
Comment (tisonkun) [inline docs/v0.9/en/user-guide/query-data/view.md:61]:
```suggestion
    ON monitor.host = app_monitor.host AND monitor.ts = app_monitor.ts
```

**Review by tisonkun — APPROVED** [review]

**Pull request merged**